docs/source/index.rst

```diff
@@ -0,0 +1,42 @@
+MaRDA Metadata Extractors Schema
+================================
+
+A repository containing the |LinkML|_-based schemas backing the MaRDA Metadata Extractors Registry and API.
```

Comment:
I don't think the sphinx page currently links to the GitHub repo anywhere, so probably add it here:

```suggestion
A repository (`marda-alliance/metadata_extractors_schema <https://github.com/marda-alliance/metadata_extractors_schema/>`) containing the |LinkML|_-based schemas backing the MaRDA Metadata Extractors Registry and API.
```